pom.xml

```diff
@@ -103,7 +103,7 @@
     <timestamp>${maven.build.timestamp}</timestamp>
     <maven.build.timestamp.format>yyyy-MM-dd HH:mm</maven.build.timestamp.format>
 
-    <beam.version>0.6.0</beam.version>
+    <beam.version>2.0.0</beam.version>
```

Comment:
I believe the dependency version numbers in this pom.xml need to be checked against the ones in the Beam root pom.xml. As a spot-check, I saw surefire here is at 2.19.1.